src/Cli/CronTaskCommand.php

```diff
@@ -18,9 +18,15 @@
  * This command finds, checks and processes all crontasks
  * Hidden because there's no reason to run this manually other than for debugging
  */
-#[AsCommand('cron-task', 'Runs cron tasks that are scheduled to be run', aliases: ['dev/cron'], hidden: true)]
+#[AsCommand('cron-task', 'Runs cron tasks that are scheduled to be run', aliases: ['dev/cron'])]
 class CronTaskCommand extends Command
 {
+    protected function configure(): void
+    {
+        parent::configure();
+        $this->addOption('debug', description: 'Enable debug output');
```

Comment:
Had to check - but mode left as default will fall back to `InputOption::VALUE_NONE` which is what we want. Normally that should be explicitly defined because it's not part of the method signature so could change, but I think it'll be fine leaving it off this time.